.github/workflows/build.yml

```diff
@@ -48,4 +48,18 @@ jobs:
         run: |
           make buildx.image
           make buildx.push
-
+        
+  trivy:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: 'docker.io/openebs/linux-utils'
```

Comment:
Dont we need to mention the tag here?

Reply:
No...it automatically fetches latest one

Reply:
Understood. It was scanning a already published docker image from docker hub. Now it scans locally generated image.